extract.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/joho/godotenv"
+)
+
+func get_etherscan_key() string {
```

Comment:
with Go, the naming convention is CamelCase (and not snake_case like for your function). see: https://go.dev/doc/effective_go#mixed-caps